client/influxdb.go

```diff
@@ -624,6 +628,17 @@ func (c *Client) Addr() string {
 	return c.url.String()
 }
 
+// checkPointTypes ensures no unsupported types are submitted to influxdb, returning error if they are found.
+func checkPointTypes(p Point) error {
+	for _, v := range p.Fields {
+		switch v.(type) {
+		case uint64:
```

Comment:
are uint64 < math.MaxInt64 supported? or do you mean we should cast to int64. personally i think it would provide a clearer API to say 'no uint64' rather than, 'uint64, providing they're less than math.MaxInt64'. Obviously up to you guys, but just my two cents.